scene/resources/label_settings.cpp

```diff
@@ -74,9 +82,11 @@ void LabelSettings::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::COLOR, "outline_color"), "set_outline_color", "get_outline_color");
 
 	ADD_GROUP("Shadow", "shadow_");
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "shadow_size", PROPERTY_HINT_RANGE, "0,127,1,or_greater,suffix:px"), "set_shadow_size", "get_shadow_size");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "shadow_outline_size", PROPERTY_HINT_RANGE, "0,127,1,or_greater,suffix:px"), "set_shadow_outline_size", "get_shadow_outline_size");
```

Comment:
This correction breaks compatibility, so I don't think this correction is appropriate

Reply:
In layman's terms, the existing property should still exist but be deprecated.

See other deprecated properties for a bit of inspiration (for example Control's `auto_translate`)